File: Fundamentos de Programación/Ejercicio 2.9.cpp
#include <iostream>

int main()
{
    int n = 0, c = 2, p = 2;
    std::cout<<"Introduce el número N: ";
    std::cin >>n;
    while(p<=n)
    {
        if(p%c == 0)
        {
            if(p == c)
            {
                std::cout<<p<<"\n";
            }
            p++;
            c = 2;
        }
        else
        {
            c++;
        }
    }
}

File: Fundamentos de Programación/Ejercicio 2.9.py
n = int(input('Introduce el número N = '))
c = 2
p = 2
while p <= n:
    if p%c == 0:
        if p==c:
            print(p)
        p = p+1
        c = 2
    else:
        c = c + 1
exit()
